project/ScalafixBuild.scala

```diff
@@ -342,4 +342,16 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
       }
     }
   )
+  private def testRulesFor(scalaVersion: String, state: State): State = {
+    supportedScalaVersionforRules(scalaVersion)
+      .filter(_ != scalaVersion)
+      .flatMap { v =>
+        List(
+          s"""set testsInput/scalaVersion := "$v"""",
+          s"""set testsOutput/scalaVersion := "$v"""",
```

Comment:
as discussed in the other thread, this does not need to be (re)-compiled, as only sources are used for assertions
```suggestion
```

Reply:
you're right .. forgot to remove it.